ext/tags.go

```diff
@@ -55,6 +65,9 @@ var (
 	// PeerService records the service name of the peer
 	PeerService = stringTagName("peer.service")
 
+	// PeerAddress records the address name of the peer
```

Comment:
I would be fine with the precise text from the table at https://github.com/opentracing/specification/blob/master/semantic_conventions.md#span-tags-table with the JDBC bit translated to the database DSN or simply removed... either way.